test/dotnet-migrate.Tests/GivenThatIWantToMigrateSolutions.cs

```diff
@@ -64,6 +64,11 @@ private void MigrateAndBuild(string groupName, string projectName)
             //    .Should().Pass();
 
             SlnFile slnFile = SlnFile.Read(Path.Combine(projectDirectory.FullName, solutionRelPath));
+
+            slnFile.ProductDescription.Should().Be("Visual Studio 15"); ;
```

Comment:
I would create a separate test for this and call it ItUpdatesTheVisualStudioVersionTo2017.